### PR TITLE
feat: add an `open` function

### PR DESCRIPTION
**Pull request by JorickPepin**
An `open` task has [recently been added](https://github.com/jolicode/docker-starter/pull/282/files#diff-b824f66b797834738d78126819926ed443d87080a25c369853ed32c1b757cb54R59) to Docker Starter. However, it works only on macOS as the Linux command is `xdg-open`. Having it here allows using the right command according to the operating system thanks to the OSHelper. WDYT?

### CODE REVIEW

File: src/functions.php [hunk 1]
```diff
@@ -1055,3 +1055,29 @@ function guard_min_version(string $minVersion): void
         throw new MinimumVersionRequirementNotMetException($minVersion, $currentVersion);
     }
 }
+
+/**
+ * @param string|non-empty-array<string> $url
+ */
+function open(string|array $url): void
```
Comment (ruudk) [inline src/functions.php:1062]: Did you consider doing
```suggestion
function open(string ...$url): void
```

Comment (lyrixx) [reply src/functions.php:1062]: I like it !

**Review by lyrixx — APPROVED** [review]
👍🏼 except a little comment

**Review by lyrixx — APPROVED** [review]
Thanks

**Pull request merged**